doc/additional_commands.rst

```diff
@@ -51,7 +51,20 @@ Universum in non-CI mode has the following differences from default mode:
 Poll chosen VCS for updates
 ---------------------------
 
-.. TODO: Add description
+This mode was created for CI systems, not having polling feature at all or exact functionality, listed below.
+
+When launched in poller mode, Universum uses a simple database to remember for every passed VCS source
```

Comment:
"...to remember latest checked commit for every passed VCS source, such as Git or P4 branch."